templates/base.html

```diff
@@ -30,11 +32,39 @@
                     {{ nav_button('contact', 'library.contact', 'Contact Us' ) }}
                     {{ nav_button('add_book', 'library_books.add_book', 'Add new book' ) }}
                     {% if session['id'] %}
-                        {{ nav_button('logout', 'library.logout', 'Logout' ) }}
                         {{ nav_button('wishlist', 'library.wishlist', 'Wish List' ) }}
-                    {% else %}
-                        {{ nav_button('login', 'library.login', 'Login' ) }}
-                        {{ nav_button('registration', 'library.registration', 'New User?' ) }}
+                            <div class="dropdown">
```

Comment:
Maybe use Jinja's macro for this dropdown?

Reply:
Oki, I can change it :)